client/src/components/FrontPage/BasicGrid.js

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid2 from '@mui/material/Unstable_Grid2';
+import CourseCard from './CourseCard';
+
+// disableEqualOverflow ?
+// is course code unique for all here?
+// xs, sm etc
```

Comment:
Comments here are a bit unclear.

Reply:
@marillam Can you clarify?

Reply:
Those were more questions than anything, currently code uses course code as unique key but should probably change when there is a unique identifier available. Rest are for grid element styling, I can delete them.